tsfile/src/main/java/org/apache/iotdb/tsfile/read/common/BatchData.java

```diff
@@ -515,6 +515,14 @@ public long getMaxTimestamp() {
     return getTimeByIndex(length() - 1);
   }
 
+  public TimeColumn getTimeSeries() {
```

Comment:
How about changing to `getTimeColumn`, as all the naming in this method

Reply:
done